apps/ejabberd/src/mod_pubsub.erl

```diff
@@ -696,6 +702,27 @@ disco_items(Host, Node, From) ->
     end.
 
 %% -------
+%% callback that prevents routing subscribe authorizations back to the sender
+%%
+
+handle_pep_authorization_response({From, To, #xmlel{ name = <<"message">> } = Packet} = Acc)
```

Comment:
It's most probably true. I have to review all hook callbacks in `mod_pubsub`. (bleh)